multicluster/controllers/multicluster/clusterset_controller.go

```diff
@@ -206,18 +206,16 @@ func (r *ClusterSetReconciler) updateMultiClusterSetOnMemberCluster(clusterSet *
 	if r.RemoteClusterManager == nil {
 		r.RemoteClusterManager = internal.NewRemoteClusterManager(r.clusterSetID, r.Log, r.clusterID)
 		go func() {
```

Comment:
@aravindakidambi I am wondering why we need a go routine here? since I noticed that there will be an inconsistent issue if r.RemoteClusterManager is initialized and used in other controller like serviceexporter controller, but after a while, it might become nil when start() failed. I suppose we can start it without go routine?